mapper/types.go

```diff
@@ -43,6 +43,16 @@ var (
 		Decimals: 18, //nolint:gomnd
 	}
 
+	Erc721DefaultCurrency = &types.Currency{
+		Symbol:   "ERC721",
```

Comment:
I think you want to pull this from the contract instead of specifying `ERC721` here. You should put that in the metadata as "Type"